epictrack-web/src/components/shared/specialField/index.tsx

```diff
@@ -311,6 +311,8 @@ export const SpecialFieldGrid = ({
         border: `2px solid ${Palette.neutral.bg.dark}`,
         background: Palette.white,
         overflowX: "auto",
+        position: "absolute",
+        zIndex: 2,

```

Comment:
why do we need zIndex?

Also zIndex is better obtained from the mui theme

const theme = useTheme()

zIndex: theme.zIndex...

Reply:
not needed will remove